research-hub-web/src/app/components/articles/articles.component.ts

```diff
@@ -52,15 +48,10 @@ export class ArticlesComponent implements OnInit {
     if (!!this.slug) {
       this.getArticleBySlug(this.slug).subscribe(data => {
         this.article$ = this.getArticleByID(data.sys.id);
-        // this.article$.subscribe(data => {
-        //   this.assets = data.body.links.assets.block;
-        //   this.inlineEntry = data.body.links.entries.inline;
-        //   this.blockEntry = data.body.links.entries.block;
-        //   this.hyperlinkEntry = data.body.links.entries.hyperlink;
-        // });
       });
       this.parentSubHubs = await this.cerGraphQLService.getParentSubHubs(this.slug);
     } else {
+      this.appComponentService.setTitle('Articles');
```

Comment:
Just realized you're not setting the title if we're loading an individual content item, if you were wondering how to get access to the title because we're using the `async pipe` instead of subscribing, you can `tap` into the observable (this won't execute until the `async` pipe is called inside the template). Here you go:

```ts
      this.getArticleBySlug(this.slug).subscribe(data => {
        this.article$ = this.getArticleByID(data.sys.id)
          .pipe(tap(res => this.appComponentService.setTitle(res.title)))
      })
```